bundles/org.openhab.binding.vesync/README.md

```diff
@@ -66,43 +66,48 @@ Channel names in **bold** are read/write, everything else is read-only
 
 ### AirPurifier Thing
 
-| Channel              | Type                 | Description                                                | Model's Supported | Controllable Values   |
-|----------------------|----------------------|------------------------------------------------------------|-------------------|-----------------------|
-| **enabled**          | Switch               | Whether the hardware device is enabled (Switched on)       | 600S, 400S, 300S  | [ON, OFF]             |
-| **childLock**        | Switch               | Whether the child lock (display lock is enabled)           | 600S, 400S, 300S  | [ON, OFF]             |
-| **display**          | Switch               | Whether the display is enabled (display is shown)          | 600S, 400S, 300S  | [ON, OFF]             |
-| **fanMode**          | String               | The operation mode of the fan                              | 600S, 400S        | [auto, manual, sleep] |
-| **fanMode**          | String               | The operation mode of the fan                              | 200S, 300S,       | [manual, sleep]       |
-| **manualFanSpeed**   | Number:Dimensionless | The speed of the fan when in manual mode                   | 600S, 400S        | [1...4]               |
-| **manualFanSpeed**   | Number:Dimensionless | The speed of the fan when in manual mode                   | 300S              | [1...3]               |
-| **nightLightMode**   | String               | The night lights mode                                      | 200S, 300S        | [on, dim, off]        |
-| filterLifePercentage | Number:Dimensionless | The remaining filter life as a percentage                  | 600S, 400S, 300S  |                       |
-| airQuality           | Number:Dimensionless | The air quality as represented by the Core200S / Core300S  | 600S, 400S, 300S  |                       |
-| airQualityPM25       | Number:Density       | The air quality as represented by the Core400S             | 600S, 400S, 300S  |                       |
-| errorCode            | Number:Dimensionless | The error code reported by the device                      | 600S, 400S, 300S  |                       |
-| timerExpiry          | DateTime             | The expected expiry time of the current timer              | 600S, 400S        |                       |
-| schedulesCount       | Number:Dimensionless | The number schedules configured                            | 600S, 400S        |                       |
-| configDisplayForever | Switch               | Config: Whether the display will disable when not active   | 600S, 400S, 300S  |                       |
-| configAutoMode       | String               | Config: The mode of operation when auto is active          | 600S, 400S, 300S  |                       |
```

Comment:
Yeah I think this was removed intentionally - so long ago I am struggling to remember now the reverse engineering of the python lib, this is based from. I will have another look at that to check if warm node is supported and understand this, while this soak tests with the adjustments. Will come back to this one, once I've had a couple of weeks (sorry works manic) to figure it all out.

Reply:
Oh just take your time. As long as you manage to be sure and we can make the use of this consistent.

Reply:
I take back the last statement I think that was a mistake after cross checking it. I have just pushed a load of minor adjustments, to clean up little bits, and added that back in. Will continue checking tomorrow, but that likely was a mistake on my part. I'll have another look tomorrow in case I can see a reason why it was removed, but at the moment I cant spot it.